rbac/server/blockchain_transactions/role_transaction.py

```diff
@@ -17,9 +17,67 @@
 import rethinkdb as r
 from rbac.server.db.db_utils import create_connection
 from rbac.common.role.delete_role_admin import DeleteRoleAdmin
+from rbac.common.role.delete_role_owner import DeleteRoleOwner
+from rbac.common.role.delete_role_member import DeleteRoleMember
 from rbac.common.sawtooth import batcher
 
 
+async def create_delete_role_member_txns(key_pair, next_id, txn_list):
+    """Create the delete transactions for a member if user is an member of any roles."""
+    conn = await create_connection()
+    roles = (
+        await r.table("role_members")
+        .filter({"related_id": next_id})
+        .coerce_to("array")
+        .run(conn)
+    )
+    conn.close()
+    if roles:
+        member_delete = DeleteRoleMember()
+        for role in roles:
+            admin_delete_message = member_delete.make(
+                signer_keypair=key_pair, related_id=next_id, role_id=role["role_id"]
+            )
+            payload = member_delete.make_payload(
+                message=admin_delete_message,
+                signer_keypair=key_pair,
+                signer_user_id=key_pair.public_key,
+            )
+            transaction = batcher.make_transaction(
+                payload=payload, signer_keypair=key_pair
+            )
+            txn_list.extend([transaction])
+    return txn_list
+
+
+async def create_delete_role_owner_txns(key_pair, next_id, txn_list):
+    """Create the delete transactions for an owner if ownership of a role(s) exists."""
```

Comment:
Please add to the docstrings the details of the passed in parameters. Refer to create_delete_role_admin_txns() function as a reference.